### ISSUE TITLE
JointJs 3.0.2  Anchor Issue in link position

### ISSUE THREAD

**Issue by KarthickeyanSekar** [clientIO/joint#1144]
i just asynchronously load the nodes from **Graph**. But, link not connecting properly in the node port. i just attached the attachment for your reference. Am already raised a issue in **Google Groups**.

![app](https://user-images.githubusercontent.com/12829801/61693983-274e2d80-ad4e-11e9-8b7d-92bdf101e209.PNG)
This is the settings for the paper. 
Source Code: 
```
var paper = new joint.dia.Paper({
            width: 2000,//"100%",
            height: 1000,//"100%",
            model: graph,
            snapLinks: {
                radius: 75
            },
            interactive: function (cellView) {
                if (cellView.model instanceof joint.shapes.standard.Link) {
                    // Disable the default vertex add functionality on pointerdown.
                    return { vertexAdd: false };
                }
                return true;
            },
            linkPinning: false,
            multiLinks: false,
            perpendicularLinks: false,
            async: true,
            sorting: joint.dia.Paper.sorting.APPROX,
            viewport: function (view, isInViewport, paper) {
                if (view != null && view.model != null && typeof view.model['getBBox'] == "function") {
                    var _cViewBBx = view.model.getBBox();
                    var _in_cViewBBx = _cViewBBx.inflate(10);
                    var is_intersect = this.windowBBox.intersect(_in_cViewBBx);
                    return is_intersect;
                } else {
                    return false;
                }
​
​
            }.bind(this)
```

> **Romans Repiled**:
> 
> HI,
> it's most likely a bug in JointJS. I am looking into it. Could you please try to use the defaultAnchor: { name: 'modelCenter' }? Is the result any better?
> 
> Thanks,
> Roman

Thanks for Support @kumilingus 
Regards
Karthikeyan Sekar

**Comment by kumilingus** [comment]
Hi @KarthickeyanSekar, could you please check again against #1145?

**Comment by kumilingus** [comment]
Fixed in `v3.0.3`.

**Comment by slavede** [comment]
I have noticed this happening in 3.2.0 again :(. I'm using RappidJS though, but under the hood I believe it has something of JointJS.

Randomly, but it happens, I'm using fromJson to load the cells, and sometimes connector are not well placed (again, it's quite random, but I see it happened before).

It seem that your workaround by setting defaultAnchor: { name: 'modelCenter' } works.

@kumilingus
